docs/source/tutorial.rst

```diff
@@ -218,3 +218,27 @@ Performance Considerations
 **************************
 If your dataset is small enough to fit in your system's memory then you can
 load it all into memory by passing ``in_memory=True`` to ``PyDssResults``.
+
+Estimate space required by ``QSTS`` simulation
```

Comment:
This can also be useful for Snapshot simulations.

Reply:
True, even it's one time point. Change ``QSTS`` to ``PyDSS``.